examples/graph.ts

```diff
@@ -0,0 +1,32 @@
+import { Graph } from 'graphlib';
```

Comment:
Please remove the examples. The correct form of these tests shall be used as tests.